frontend/src/metabase/lib/formatting/value.tsx

```diff
@@ -177,7 +177,7 @@ export function formatValueRaw(
   } else if (isEmail(column)) {
     return formatEmail(value as string, options);
   } else if (isTime(column)) {
-    return formatTime(value as Moment);
```

Comment:
We should delete the old `formatTime` entirely I don't think it's used anywhere else, except the static viz `format.ts`, which we probably want to replace with `formatTimeWithOptions` as well

<img width="631" alt="Screenshot 2024-01-19 at 4 11 39 PM" src="https://github.com/metabase/metabase/assets/37751258/25a80112-c2a3-46c0-8f21-c4db4ba3933c">